win32ss/user/ntuser/ntstubs.c

```diff
@@ -783,7 +783,7 @@ NtUserSetInformationThread(IN HANDLE ThreadHandle,
             Status = STATUS_SUCCESS;
             _SEH2_TRY
             {
-                ProbeForRead(ThreadInformation, sizeof(CsrPortHandle), sizeof(PVOID));
+                ProbeForRead(ThreadInformation, sizeof(CsrPortHandle), _alignof(CsrPortHandle));
```

Comment:
single-underscore alignof?
Note that we also have:
[#define TYPE_ALIGNMENT(t) __alignof(t)](https://git.reactos.org/?p=reactos.git;a=blob;f=sdk/include/xdk/ntbasedef.h;hb=be8a6f8af19b7b9f7be29ea6b81ed8da029daa2c#l113)
acts on a type though.

Reply:
Fixed using the double underscore version.